plaid/date.go

```diff
@@ -0,0 +1,11 @@
+package plaid
+
+import "time"
+
+// DateLayout is the time format expected by the api
+const DateLayout = "2012-11-01"
+
+// Date returns a time from the specified date
+func Date(year int, month time.Month, day int) time.Time {
```

Comment:
This function is likely superfluous seeing as we don't want to reimplement time and this is only used in tests.

Reply:
@cbelsole It's intended to be used by the client. I found that using `time.Date` was very clunky with all the zero values. Also, the limited parameters serve as implicit documentation that these are the only fields that are actually used in the request.